docs/intro/arch_overview/tracing.rst

```diff
@@ -45,6 +31,44 @@ initiated:
 * Randomly sampled via the :ref:`random_sampling <config_http_conn_man_runtime_random_sampling>`
   runtime setting.
 
+Trace context propagation
+-------------------------
+Envoy provides the capability for reporting tracing information regarding communications between
+services in the mesh. However, to be able to correlate the pieces of tracing information generated by the
+various proxies within a call flow, the services must propagate certain trace context between the ingress
+and egress requests.
+
+Whichever tracing provider is being used, the service should propagate the
+:ref:`config_http_conn_man_headers_x-request-id` to enable logging across the invoked services
+to be correlated.
+
+The tracing providers also require additional context, to enable the parent/child relationships
+between the spans (logical units of work) to be understood.
+
+When using the LightStep tracer, Envoy relies on the service to propagate the
+:ref:`config_http_conn_man_headers_x-ot-span-context` HTTP header
+while sending HTTP requests to other services.
+
+When using the Zipkin tracer, Envoy relies on the service to propagate either the
+official B3 HTTP headers (
+:ref:`config_http_conn_man_headers_x-b3-traceid`,
+:ref:`config_http_conn_man_headers_x-b3-spanid`,
+:ref:`config_http_conn_man_headers_x-b3-parentspanid`,
+:ref:`config_http_conn_man_headers_x-b3-sampled`, and
+:ref:`config_http_conn_man_headers_x-b3-flags`) or for convenience it is
+also possible to just propagate the
+:ref:`config_http_conn_man_headers_x-ot-span-context` HTTP header.
+
+NOTE: Work is currently underway in the distributed tracing community to define a standard
+for trace context propagation. Once a suitable approach has been adopted, the use of the non-standard
+single header :ref:`config_http_conn_man_headers_x-ot-span-context` will be replaced.
+
+Propagation of the trace context can also be achieved by using the LightStep or Zipkin tracer directly
+within the service itself, to extract the trace context from the ingress request and
+inject it into any subsequent egress requests.
```

Comment:
Nit: inbound and outbound instead of ingress vs egress. But if you feel that this correlates better with the ingress and egress operations, then leave it be.